### ISSUE TITLE
Machine Learning Modell trainieren (Noise Classifier)

### ISSUE THREAD

**Issue by JeffyPayos** [JeffyPayos/Sensorik#7]
Ein Modell entwickeln, das erkennt, ob ein Punkt „Noise“ oder „Objekt“ ist.

✔️ Aufgaben

Modell wählen:

RandomForestClassifier (Start)

oder kleines Neural Network (PyTorch / TensorFlow)

Train/Test Split (z. B. 80/20)

Training starten

Evaluierung:

Accuracy

Precision

Recall

F1 Score

Confusion Matrix

Modell speichern:

models/noise_filter.pkl oder .pt
